build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -107,6 +107,9 @@ public override string ToString()
         //
         public bool IsLocked = true; // Set this variable to toggle muxer locking.
         public VerInfo LockedHostFxrVersion => IsLocked ? new VerInfo(1, 0, 1, "", "", "", CommitCountString) : LatestHostFxrVersion;
+        //We add a different LockedHostFxrVersion for the MSI as the package needs to deviate for issue #1574. This we can update or remove when we have a binary change for the hostfxr.dll (from 1.0.1)
+        public bool fExplicitHostFXRMSIVersion = true; // Set this variable to override LatestHostFxrVersion for MSI packages only. This can be set to false when we fall back to using just LockedHostFxrVersion for all.
```

Comment:
The last sentence should be something like "This should be set to false when we no longer need to override the MSI version to be different from the HostFXR nuget package version".